src/ucs/sys/sock.c

```diff
@@ -334,11 +335,18 @@ ucs_status_t ucs_socket_send(int fd, const void *data, size_t length,
                               "send", err_cb, err_cb_arg);
 }
 
+/* recv is declared as 'always_inline' on some platforms, it leads to
+ * compilation warning. wrap it into static function */
+static ssize_t ucs_socket_recv_io(int fd, void *data, size_t size, int flags)
```

Comment:
is it possible to suppress this warning somehow instead?

Reply:
specific command line option to suppress warning... but this is not good idea

Reply:
it is used in `uct_tcp_ep_recv` which is invoked by TCP progress
should not affect perf if `recv` is declared inline though